cli/command/container/cmd.go

```diff
@@ -7,12 +7,13 @@ import (
 )
 
 // NewContainerCommand returns a cobra command for `container` subcommands
+// nolint: interfacer
 func NewContainerCommand(dockerCli *command.DockerCli) *cobra.Command {
```

Comment:
There are some auth methods that don't exist on the interface yet. And if I change this I have to change everything in the `container/` package, so I wanted to delay that change.

Most `// nolint` should be considered the equivalent of a `// TODO: fix this`